### DISCUSSION TITLE
Third party download from CurseForge

### DISCUSSION THREAD

**Discussion by Celivalg** [NovaMachina-Mods/ExNihiloSequentia#349] [Q&A] [answered]
Hello! I would like some clarification on this topic, on if that was a mistake or if it is meant to be that way.

You blocked third party downloads on CurseForge or something, this is due to the new CurseForge API that allows a content creator to do that. Here are a few screenshots from installing ATM7 To The Sky in PolyMC: 
![expolymc1](https://user-images.githubusercontent.com/32540922/186756459-c8945c82-ae7c-4094-8ecf-4fd0b4029a73.png)
![expolymc2](https://user-images.githubusercontent.com/32540922/186756469-d884b14f-7e44-4781-9417-e278dabe75a2.png)

This is unrelated to #346 as far as I can tell.

Whether you want to keep it that way or not is up to you. But as a member of the linux community, we don't have a way to get mod-packs automatically otherwise... 

Thank you

**Comment by NovaMachina** [comment, marked as answer]
At the current moment, yes this is the intended behavior. Currently I am waiting to see if there is any further development from CurseForge about how they will handle Third-party downloads and the linux community. As someone who daily drives Fedora, I understand the frustration and there is no intention of excluding anyone. However, at the current time the number of people who appear to be effected by the current CurseForge policy seems to be very low. But if that changes in the future and I see a significant increase in the number of people who are running into this particular issue, then I will reconsider the current settings.

**Comment by AmbroseSilverWolfe** [comment]
I came here to ask about the same thing. I use a free server called Aternos (.org). The site does not allow manually adding mods to the server. It has its own "library" that is pulls from curseforge (to prevent users from adding malicious content) and allows users to add mods that way. Unfortunately, Ex Nihilo: Sequentia is unable to be added to their list due to "Not allowing Project Distribution"
